iphone/Classes/TiUIListView.m

```diff
@@ -2012,7 +2012,7 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView
         TiUIListSectionProxy *section;
         CGFloat topSpacing = scrollView.contentOffset.y + scrollView.adjustedContentInset.top;
 
-        NSString *direction = nil;
+        NSString *direction = [NSNull null];
```

Comment:
Was the typing okay or did Xcode complain?

Reply:
I didn't check it with Xcode, I've only copied it with a normal text editor, build the SDK and checked it in an app :smile: